readme.md

```diff
@@ -0,0 +1,103 @@
+# Hello Microverse<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+-
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello-Microverse Project <a name="about-project"></a>
+
+**Hello-Microverse** is an introductory project to easily show the use of basic programming languages
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> css and html
+
+### Key Features <a name="key-features"></a>
+
+- **Html & css**
+- **Linters config**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+follow the below steps to get started
+
+### Prerequisites
+
+>you need to have the knowledge of git and github commands before this project
+
+### Setup
+
+>clone the folder in your specified directory by "git clone + link" and make sure the file has be copied.
+
+
+### Install
+
+> use Visual studio to the project to install the project by pull request
+
+### Usage
+
+>You can run it with commads on git that include git add . ,git commit and git push
+
+### Run tests
+
+>open the project after you made whole changes
+
+### Deployment
+request for the code review
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Tsige Wossne**
+
+- GitHub: [@githubhandle](https://github.com/tsigewossne)
+- Twitter: [@twitterhandle]
+- LinkedIn: [LinkedIn]
```

Comment:
- [ ] Kindly add your twitter and LinkedIn accounts as well. It will help readers to reach you easily.